report/unsafe/sections/albin_arrowhead_communication.tex

```diff
@@ -0,0 +1,92 @@
+\subsection{Arrowhead framework}
+Arrowhead framework is a framework designed to aid in the functonality of Iot-devices, havning a focus on large scale industry and
+productivity. The framework provides a number of core system, systems that are required to run to enable certain functionality. This 
```

Comment:
"...a number of core systemS that are reqyired to run AND enable..." I think it sound better